docs/rule/no-bare-strings.md

```diff
@@ -30,10 +30,25 @@ This rule **allows** the following:
 
 When the config value of `true` is used the following configuration is used:
 
-* `whitelist` - `(),.&+-=*/#%!?:[]{}`
+* `whitelist` - refer to the `DEFAULT_CONFIG.whitelist` property in the [rule](../lib/rules/no-bare-strings.js)
```

Comment:
```suggestion
* `allowedList` - refer to the `DEFAULT_CONFIG.allowedList` property in the [rule](../lib/rules/no-bare-strings.js)
```

While we're making these changes, let's go ahead and improve on the terms (as related to issue #989 )

Reply:
I'm happy to pick up #989 (in its entirety!). Any objections to doing that as a separate MR? I feel like renaming config keys might end up being a breaking change, whereas this work would not be.

Reply:
Yeah let's do that separately since it's a breaking change.